stdpopsim/catalog/AraTha/genome_data.py

```diff
@@ -0,0 +1,14 @@
+# File autogenerated from Ensembl REST API. Do not edit.
+data = {
+    "assembly_accession": "GCA_000001735.1",
+    "assembly_name": "TAIR10",
+    "chromosomes": {
+        "1": {"length": 30427671, "synonyms": []},
```

Comment:
I don't see any synonyms here or in any of the other files? That means something is missing right?

Reply:
Actually I do see them in human. Do none of the other species have synonyms?

Reply:
No, looks like Ensembl's list of synonyms is pretty limited. We could bug them about this, I guess.

Reply:
I think we kinda have to, otherwise the synonyms aren't actually that useful. If they could at least add the ones that we've already found that would be great. Also I don't see us having a  `list_chromosomes()` function implemented. That would allow us to get the user to get things in the correct format for us as they could easily see the formatting themselves.

Reply:
strikes me that we could easily add some default synonyms to each species e.g., 1, chr1, Ch1

Reply:
Moving discussion to #570